src/components/form/FormContainer.jsx

```diff
@@ -0,0 +1,189 @@
+						</Paragraph>
+					) : (
+						<Paragraph variant='body3' className={'text-primary ' + item.classes} key={index}>
+							{item.text}
+						</Paragraph>
+					)
+				)}
+				<div id='imgContainer' className='mx-auto my-8 w-[50vw] md:w-[408px]'>
+					<img src={formimg} height={'100%'} width={'100%'} alt='ticket-price' />
+				</div>
+				<Paragraph variant='body3'>
+					Alumni donating/contributing more than ₹50,000 can pay &quot;Individual Only (without family)&quot; registration
+					fee for their family registration and they do not need to pay &quot;With Family (spouse + kids)&quot; registration
+					fee.
+				</Paragraph>
+
+				<Heading
+					variant='h3'
+					className='text-primary-yellow  mt-10 underline'
+					style={{
+						textShadow: ' 1px 1px 0px #F9F9F9',
+						WebkitTextStrokeWidth: 0.8,
+						WebkitTextStrokeColor: '#252525',
+					}}>
+					The Registration fee covers the following
+				</Heading>
+
+				<ul className='list-disc mx-3 my-5'>
+					{feeCoverage.map((item, index) => (
+						<li key={index}>
+							<Paragraph variant='body3' key={index}>
+								{item}
+							</Paragraph>
+						</li>
+					))}
+				</ul>
+
+				<Paragraph variant='body2' className='text-primary my-2'>
+					The fee does not include accommodation.
+				</Paragraph>
+			</div>
+
+			<div id='form' className='w-[90%] md:w-[55%] m-8 p-2'>
+				{inputContent.map(item => (
+					<React.Fragment key={item.key}>
+						<Paragraph
+							variant='body2'
+							htmlFor={item.id}
+							className=' shadow-white px-2'
+							style={{
+								textShadow: ' 1px 1px 0px #FF7342',
+								WebkitTextStrokeWidth: 0.5,
+								WebkitTextStrokeColor: '#252525',
+							}}>
+							{item.label}
+						</Paragraph>{' '}
+						{Array.isArray(item.id) ? (
+							item.id.map((id, idx) => (
+								<Inputs
+									key={idx}
+									className='inline mr-3 w-[31.3%]'
+									onChange={e => setInputValue(id, e.target.value, e.target.validated)}
+									validated={false}
+									formData={{
+										type: item.type[idx],
+										minLength: item.minLength[idx],
+										maxLength: item.maxLength[idx],
+										regex: item.regex[idx],
+										id: id,
+										placeholder: item.placeholder[idx],
+									}}
+								/>
+							))
+						) : (
+							<Inputs
+								className='block w-[98%]'
+								onChange={e => setInputValue(item.id, e.target.value, e.target.validated)}
+								validated={false}
+								formData={{
+									type: item.type,
+									minLength: item.minLength,
+									maxLength: item.maxLength,
+									regex: item.regex,
+									id: item.id,
+									placeholder: item.placeholder,
+								}}
+							/>
+						)}
+					</React.Fragment>
+				))}
+
+				<textarea
+					className='outline-none	bg-inherit rounded-md my-3 mb-6 w-[98%] h-[8vw] text-[#B0B0B0] p-2 block'
+					style={{ border: '1px solid #FF7647', boxShadow: '2px 2px 0px 0px #F9F9F9' }}
+					placeholder='Registration Type'
```

Comment:
Wrap textArea into a new component and the style the component and use that component over here

Reply:
done. https://github.com/dscnitrourkela/project-udon/pull/11/commits/955977ced6d1d2c4f9a63e243e57a8ab298405bb